eldritch/encounters.py

```diff
@@ -252,11 +252,134 @@ def Graveyard6(char):
+  return events.Sequence([die, stamina], char)
+def House4(char):
+  stay = events.Delayed(char)
+  common = events.Nothing() # TODO: coming soon from Peter! draw and purchase
+  unique = events.Nothing() # TODO: coming soon from Peter! draw and purchase 
+  item = events.BinaryChoice(char, "Purchase a common or unique item?", "Common", "Unique", common, unique)
+  will = events.Check(char, "will", 0)
+  converse = events.PassFail(char, will, item, stay)
+  return events.BinaryChoice(char, "Converse with travelling Salesman?", "Yes", "No", converse, events.Nothing())
+def House5(char):
+  return events.Draw(char, "common", 1)
+def House6(char):
+  luck = events.Check(char, "luck", -1)
+  stamina = events.Gain(char, {"stamina": 1})
+  sanity = events.Gain(char, {"sanity": 1})
+  choice = events.BinaryChoice(char, "Gain 1 stamina or sanity?", "Stamina", "Sanity", stamina, sanity)
+  return events.PassFail(char, luck, choice, events.Nothing())
+def House7(char):
+  prereq = events.AttributePrerequisite(char, "dollars", 3, "at least")
+  pay = events.Loss(char, {"dollars": 3})
+  gain = events.SplitGain(char, "sanity", "stamina", 4)
+  stay = events.Sequence([pay, gain], char)
+  choice = events.BinaryChoice(char, "Spend $3 to spend the night?", "Yes", "No", stay, events.Nothing())
+  return events.PassFail(char, prereq, choice, events.Nothing())
+
+def Church1(char):
+  return events.Loss(char, {"sanity": 1})
+def Church2(char):
+  return events.Bless(char)
+def Church3(char):
+  money = events.Loss(char, {"dollars": (char.dollars + 1)//2})
+  items = events.Nothing() # TODO: Lose half your items
+  return events.BinaryChoice(char, "Donate half or your money or half of your items.","Money", "Items", money, items)
+def Church4(char):
+  holywater = events.DrawSpecific(char, "unique", "Holy Water")
+  return events.BinaryChoice(char, "Search for Holy Water?", "Yes", "No", holywater, events.Nothing())
+def Church5(char):
+  check = events.Check(char, "luck", 0)
+  loseSanity = events.Loss(char, {"sanity": 3})
+  move = events.ForceMovement(char, "Southside")
+  gainSanity = events.Gain(char, {"sanity": char.max_sanity})
+  cond =  events.Conditional(char, check, "successes", {0: loseSanity, 1: move, 2: gainSanity})
+  return events.Sequence([check, cond], char)
+def Church6(char):
+  #TODO: remove a counter from the doomtrack
+  prereq = events.AttributePrerequisite(char, "clues", 1, "at least")
+  clue = events.Loss(char, {"clues": 1})
+  roll = events.Nothing() #TODO: roll for a success
+  doom = events.Nothing() #TODO: remove a doom token
+  chance = events.Sequence([clue, roll, doom], char)
+  choice = events.BinaryChoice(char, "Spend a clue token for a chance to remove a Doom Token?", 
+                               "Yes",
+                               "No",
+                               chance,
+                               events.Nothing())
+  return events.PassFail(char, prereq, choice, events.Nothing())
```

Comment:
Also can change this to adding a prerequisite to the BinaryChoice.

Reply:
updated